tests/hardhat/Governance/GovernanceBravo/intializeTests.ts

```diff
@@ -0,0 +1,87 @@
+import { Signer } from "ethers";
+import { ethers } from "hardhat";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { smock, MockContract, FakeContract } from "@defi-wonderland/smock";
+import chai from "chai";
+const { expect } = chai;
+chai.use(smock.matchers);
+
+import {
+	GovernorBravoDelegate, GovernorBravoDelegate__factory, XVSVault
+} from "../../../../typechain";
+
+let root: Signer;
+let customer: Signer;
+let accounts: Signer[];
+let governorBravoDelegate: MockContract<GovernorBravoDelegate>;
+let xvsVault: FakeContract<XVSVault>;
+
+type GovernorBravoDelegateFixture = {
+	governorBravoDelegate: MockContract<GovernorBravoDelegate>,
+	xvsVault: FakeContract<XVSVault>
+}
+
+async function governorBravoFixture(): Promise<GovernorBravoDelegateFixture> {
+	const GovernorBravoDelegateFactory = await smock.mock<GovernorBravoDelegate__factory>("GovernorBravoDelegate");
+	const governorBravoDelegate = await GovernorBravoDelegateFactory.deploy();
+	const xvsVault = await smock.fake<XVSVault>("XVSVault");
+	return {governorBravoDelegate, xvsVault}
```

Comment:
Let's use two spaces for indentation in new ts files pls

Reply:
Could add a githook to start linting files as they are committed to gradually get to a correct baseline. It shouldn't be necessary to leave style comments imo